src/User.php

```diff
@@ -384,11 +385,20 @@ public function roles()
      * </span>
      * ```
      *
+     * @example
```

Comment:
Do you want to add your other examples that you added in the *Usage* section of your pull request here as well?

```twig
{% if user.can('edit_term', term.id) %}
    {# do something with privileges #}
{% endif %}

{% if user.can('edit_user', user.id) %}
    {# do something with privileges #}
{% endif %}

{% if user.can('edit_comment', comment.id) %}
    {# do something with privileges #}
{% endif %}
```

Reply:
Resolved in #2647